quint/src/names/base.ts

```diff
@@ -100,6 +100,28 @@ export function copyNames(
   return table
 }
 
+/**
+ * Add namespaces to a definition's `namespaces` field, if it doesn't already
+ * have them on the latest position or in the beginning of its name.
```

Comment:
I think I meant latest in the sense that is was the last one to be added to the field. In this case, it is the last. If the field was a stack (LIFO), this would be the first. But yeah, `last` is clearer, sorry about the confusion :sweat_smile: